subnet/local/build_local_img.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+cd "$(dirname "$0")"
+git clone https://github.com/XinFinOrg/XDC-Subnet
```

Comment:
what if i already have the repo in cloned and would like to test out a local branch only?
As a bonus, we can probably test out if the `XDC-Subnet` already exist. if not, clone it. otherwise just continue with line 4

Reply:
Included suggestion and added branch option